nvflare/private/fed/app/deployer/server_deployer.py

```diff
@@ -106,9 +106,9 @@ def deploy(self, args):
         services.deploy(args, grpc_args=first_server, secure_train=self.secure_train)
 
         job_runner = JobRunner(workspace_root=args.workspace)
-        workspace = Workspace(args.workspace, "server", args.config_folder)
+        workspace = Workspace(args.workspace, SiteType.SERVER, args.config_folder)
         run_manager = RunManager(
-            server_name=services.project_name,
```

Comment:
what is "services.project_name"
could user change their server name now? or it has to be "server"

Reply:
we never used the "project_name" before. It should be properly changed to use "server".